src/modify_menu_panel.js

```diff
@@ -6,25 +6,40 @@ function ModifyMenuPanel(editor) {
 	this.menus = {};
 	this.menuColumn = null;
 	this.menuContainer = null;
-	
+
 }
 
 ModifyMenuPanel.prototype.initialize = function (parentContainer) {
 	this.menuColumn = $("<div/>").attr('class', menuColumnClass).appendTo(parentContainer);
-	
+
 	// Generate the document status panel, which shows a save/export button as well as if there are changes to the document
 	if (this.editor.options.enableDocumentStatusPanel) {
-		var documentStatusPanel = $("<div>");
+		var self = this;
+		var documentStatusPanel = $(self.editor.options.documentStatusPanelDomId);
 		$("<span/>").addClass(submissionStatusClass).html("Document is unchanged")
 			.appendTo(documentStatusPanel);
-		var submitButton = $("<input/>").attr({
-			'id' : submitButtonClass,
-			'type' : 'button',
-			'class' : 'send_xml',
-			'name' : 'submit',
-			'value' : 'Submit Changes'
-		}).appendTo(documentStatusPanel);
-		if (this.editor.options.ajaxOptions.xmlUploadPath == null) {
+
+		if (self.editor.options.ajaxOptions.xmlUploadConfig != null){
+			$.each(self.editor.options.ajaxOptions.xmlUploadConfig, function(index, el){
```

Comment:
Yes, it allows for any number of submit buttons. Our app lets multiple users edit metadata records, saving their progress at any point to a database and sending the final records to a Fedora instance  (via an API call to another app on another server) when they're ready. So our users have a very clear distinction between saving something, which lets another user pick up where they left off, and publishing it. We needed a more robust option than the export for in-progress files, it's part of their normal workflow, with many sets of records being underway at any given time.

I think a dropdown would muddy our distinction between saving and publishing, and bury an important part of the app a little bit. But you do raise a good point, which is how many submit options are people realistically going to need, and it's possible that just one extra one is enough - it is for us. 

Here are a few screenshots, let me know what you think. 

![unchanged](https://cloud.githubusercontent.com/assets/186452/6811125/c3ea9606-d233-11e4-8b58-a68ca2170184.png)

![saved](https://cloud.githubusercontent.com/assets/186452/6811131/cd6a6440-d233-11e4-900c-1b9712284301.png)

![errors](https://cloud.githubusercontent.com/assets/186452/6811134/d5b7df6a-d233-11e4-8d45-ce4c2ef76d36.png)